New Left Review.js

```diff
@@ -225,23 +205,19 @@ var testCases = [
 				"title": "Domesticating Hegel",
 				"creators": [
 					{
-						"lastName": "Michael Lipkin",
-						"creatorType": "author",
-						"fieldMode": 1
+						"firstName": "Michael",
+						"lastName": "Lipkin",
+						"creatorType": "author"
 					}
 				],
 				"date": "2021-04-30",
 				"abstractNote": "Michael Lipkin on Klaus Vieweg, Hegel: Der Philosoph der Freiheit. The author of Philosophy of Right as roistering liberal.",
 				"issue": "128",
-				"journalAbbreviation": "NLR",
+				"journalAbbreviation": "New Left Rev",
 				"libraryCatalog": "New Left Review",
 				"pages": "153-160",
 				"publicationTitle": "New Left Review",
-				"attachments": [
-					{
-						"mimeType": "application/pdf"
```

Comment:
Where'd these PDFs go?

Reply:
The download is paywalled. Do we want tests for unpaywalled or paywalled conditions?

Reply:
Paywalled is fine. Better to test that we're getting the right data, even if only some people can test it.

Reply:
I see! Thx. Retested under both conditions and PDFs are being downloaded when unpaywalled.